include/eve/module/bessel/detail/evaluate_rational.hpp

```diff
@@ -42,7 +42,7 @@ reverse_evaluate_rational(const T num, const U den, V z) noexcept
     z      = rec(z);
     auto r = reverse_horner(z, num) / reverse_horner(z, den);
     if( size(den) == size(num) ) return r;
-    else return r * pow(z, size(num) - size(den));
+    else return r * pow(z, (std::uint64_t) size(num) - size(den));
```

Comment:
this pops up because size_t is not passing the concept. And going to pow is not working.